### ISSUE TITLE
telegram-desktop crashes on Sway Wayland

### ISSUE THREAD

**Issue by shnaps** [void-linux/void-packages#51180]
### Is this a new report?

Yes

### System Info

Void 6.9.8_1 x86_64 GenuineIntel uptodate rrmFFFFFFFFFFFF

### Package(s) Affected

telegram-desktop-5.1.7_2 

### Expected behaviour

Telegram doesn't crash time from time

### Actual behaviour

Telegram crashes for no reason. It takes somewhere from 2 to 10 minutes to reproduce it. 

>telegram-desktop
QPainter::begin: Paint device returned engine == 0, type: 2
QWidget::render: Cannot render with an inactive painter
qt.svg: Error while inflating gzip file: SVG format check failed
qt.svg: Error while inflating gzip file: SVG format check failed
qt.gui.imageio.jpeg: Corrupt JPEG data: premature end of data segment
qt.gui.imageio.jpeg: Corrupt JPEG data: premature end of data segment
qt.gui.imageio.jpeg: Corrupt JPEG data: premature end of data segment
qt.gui.imageio.jpeg: Corrupt JPEG data: premature end of data segment
qt.svg: Error while inflating gzip file: SVG format check failed
qt.gui.imageio.jpeg: Corrupt JPEG data: premature end of data segment
qt.gui.imageio.jpeg: Corrupt JPEG data: premature end of data segment
qt.gui.imageio.jpeg: Corrupt JPEG data: premature end of data segment
qt.gui.imageio.jpeg: Corrupt JPEG data: premature end of data segment
qt.gui.imageio.jpeg: Corrupt JPEG data: premature end of data segment
qt.svg: Error while inflating gzip file: SVG format check failed
qt.svg: Error while inflating gzip file: SVG format check failed
[libvpx-vp9 @ 0x7fdfa02e7800] v1.13.1
[libvpx-vp9 @ 0x7fdfa02e7800] v1.13.1
qt.svg: Error while inflating gzip file: SVG format check failed
qt.svg: Error while inflating gzip file: SVG format check failed
[libvpx-vp9 @ 0x7fdfa02e8200] v1.13.1
[libvpx-vp9 @ 0x7fdfa02e8200] v1.13.1
qt.gui.imageio.jpeg: Corrupt JPEG data: premature end of data segment
qt.gui.imageio.jpeg: Corrupt JPEG data: premature end of data segment
  OpenType support missing for "Open Sans", script 11
  OpenType support missing for "vazirmatn ui nl", script 11
xdg_surface@118: error 1: xdg_popup has no parent
The Wayland connection experienced a fatal error: Protocol error

### Steps to reproduce

1. Open telegram-desktop
2. Browse some channel
3. Telegram will crash in 2-10 minutes

**Comment by classabbyamp** [comment]
does `xbps-pkgdb -a` show any errors? if so `xbps-install -f` those packages

**Comment by shnaps** [comment]
> does `xbps-pkgdb -a` show any errors? if so `xbps-install -f` those packages

Nope, nothing related. Missing config in cups-browsed and some symlink troubles with neovim.

**Comment by pedrocorral** [comment]
Same problem here with Telegram on Debian (no special configuration at all, just out-of-the-box installation).

@classabbyamp, if you want a channel that systematically breaks down the App this is: "Tech & Leaks Zone". They have posted something recently that I cannot see at all. The rest of the channels do not break it.

I hope that helps. Best regards.

**Comment by shnaps** [comment]
Going to close the issue because of the age + I moved out of linux a bit ago
